lib/tasks/rake_arguments.rb

```diff
@@ -0,0 +1,9 @@
+class RakeArguments
+  def self.task_options
+    ARGV.drop_while { |e| e != '--' }.drop(1).join(' ')
+  end
+
+  def self.more_tasks?
+    ARGV.take_while { |e| e != '--' }.drop(1).any?
```

Comment:
I'd like to see this method just return an array of arguments.
Let whoever cares about them being `any?` do that themselves.